### DISCUSSION TITLE
Apprise Payload Truncated

### DISCUSSION THREAD

**Discussion by vmsman** [netalertx/NetAlertX#719] [Q&A]
What is the recommended Max payload size for Apprise?  The default is 1024 and I am seeing the (text was truncated) message.   What is the format of the messages that NetAlertX sends?  I assumed they would be things like node discovered or node down messages.

**Comment by jokob-sk** [comment]
Hi @vmsman, 

You can always check your past notifications in Monitoring -> Sent Reports and you can switch between the message types (JSON/TEXT/HTML)

![image](https://github.com/jokob-sk/NetAlertX/assets/96159884/133f5764-e86d-41a6-b2a3-df71afbe7675)

As describe in settings: 

> Select the payoad type sent to Apprise. For example html works well with emails, text with chat apps, such as Telegram.
